From 5b78c0776c727760062fc099884b556a2186cb3c Mon Sep 17 00:00:00 2001
From: Patryk Kups <pkups@anaconda.com>
Date: Thu, 20 Nov 2025 09:25:11 +0100
Subject: [PATCH] patch

---
 tests/conftest.py | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/tests/conftest.py b/tests/conftest.py
index 9073a38..3b1e2db 100644
--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -21,12 +21,12 @@ from ._loop import LoopSocket
 from ._stub_sftp import StubServer, StubSFTPServer
 from ._util import _support
 
-from icecream import ic, install as install_ic
+# from icecream import ic, install as install_ic
 
 
 # Better print() for debugging - use ic()!
-install_ic()
-ic.configureOutput(includeContext=True)
+# install_ic()
+# ic.configureOutput(includeContext=True)
 
 
 # Perform logging by default; pytest will capture and thus hide it normally,
-- 
2.45.2

